### PR TITLE
Added deployed backend URL

### PR DESCRIPTION
**Pull request by AnujaBujurge29**
- solved minor bugs
-

### CODE REVIEW

File: src/Pages/Services/FetchAPI.jsx [hunk 1]
```diff
@@ -2,10 +2,10 @@ import { useEffect } from 'react';
 
 const FetchAPI = ({ setServicesSample, searchQuery }) => {
 
-    // const API_URL = 'https://api.sampleapis.com/cartoons/cartoons2D';
-    const API_URL = "http://localhost:8000/api/v1/vendors"
+    const API_URL = "https://gg-pac-team5-back-1.onrender.com/api/v1/vendors"
```
Comment (Pisotski) [inline src/Pages/Services/FetchAPI.jsx:5]: please import API from config file.
we keep it in .env file. If server host will change, your component will break again

import { API } from "../../config";

const API_URL=`${API}/services/vendorId/${vendorId}`

File: src/Pages/Services/Cards.jsx [hunk 2]
```diff
@@ -22,12 +22,21 @@ const Cards = () => {
...
+        5: "/Home/carousel_backup_5.jpg",
+    };
+
     useEffect(() => {
         fetchInitialData();
     }, []);
 
     const fetchInitialData = async () => {
-        const API_URL = "http://localhost:8000/api/v1/vendors";
+        const API_URL = "https://gg-pac-team5-back-1.onrender.com/api/v1/vendors"
```
Comment (Pisotski) [inline src/Pages/Services/Cards.jsx:38]: same here

**Review by Pisotski — CHANGES_REQUESTED** [review]

File: src/Pages/Services/FetchAPI.jsx (hunk 1 above)
Comment (AnujaBujurge29) [reply src/Pages/Services/FetchAPI.jsx]: Resolved

File: src/Pages/Services/Cards.jsx (hunk 2 above)
Comment (AnujaBujurge29) [reply src/Pages/Services/Cards.jsx]: Resolved

**Review by Pisotski — APPROVED** [review]

**Pull request merged**